### PR TITLE
Upgraded Go and bingo.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: .bingo/variables.env [hunk 1]
```diff
@@ -8,11 +8,17 @@ if [ -z "$GOBIN" ]; then
...
-JB="$GOBIN/jb-v0.4.0"
+CONTROLLER_GEN="${GOBIN}/controller-gen-v0.4.1"
 
-JSONNET="$GOBIN/jsonnet-v0.16.0"
+GOJSONTOYAML="${GOBIN}/gojsontoyaml-v0.0.0-20200602132005-3697ded27e8c"
 
-JSONNETFMT="$GOBIN/jsonnetfmt-v0.16.0"
+JB="${GOBIN}/jb-v0.4.0"
+
+JSONNET="${GOBIN}/jsonnet-v0.16.0"
```
Comment (paulfantom) [inline .bingo/variables.env:19]: note: there is 0.17.0 with some performance improvements ;)

Comment (bwplotka) [reply .bingo/variables.env:19]: Yup, let's not overcrowd this PR though (:

**Comment by bwplotka** [comment]
I also upgraded controller-gen accidentally. Will this work? 🤔

**Comment by bwplotka** [comment]
Thanks for review, let's actually do it properly with operate first approach (: (but also without too much overhear). 


I think I will propose bit fresher approach with operator (using our deployments jsonnet + locutus + proto entrypoint) inside deployments GitHub. I know we moved the operator to a separate repo for a reason, but I think given we have to have  close control and allow our deployments to work and be configured the same on both operator and non operator envs we have to keep them close IMO

Still I will copy most of the stuff from this repo really (:

**Pull request closed without merging**